search/search_test.go

```diff
@@ -625,6 +625,103 @@ func TestPutBadStatus(t *testing.T) {
 	}
 }
 
+func TestPutMultiNilIDSlice(t *testing.T) {
+	index, err := Open("Doc")
+	if err != nil {
+		t.Fatalf("err from Open: %v", err)
+	}
+
+	c := aetesting.FakeSingleContext(t, "search", "IndexDocument", func(in *pb.IndexDocumentRequest, out *pb.IndexDocumentResponse) error {
+		if len(in.Params.GetDocument()) < 1 {
+			return fmt.Errorf("expected at least one Document, got %v", in)
```

Comment:
Change this to `got` before `want` style as you have below. `got %v, want at least one Document`. Same throughout the rest of the tests as well for consistency.